tools/storyteller/StorytellerUiTool.java

```diff
@@ -84,28 +84,39 @@
   private JComboBox projectDropDown;
   private JTextField storyTextArea;
   private Mode mode;
+  private StorytellerConfig storytellerConfig;
 
   enum Mode {
     TIME_AND_SCREENSHOT,
     SCREENSHOT_ONLY
   }
 
+  @Singleton
+  @Component(modules = { CommonModule.class })
+  interface StorytellerUiToolComponent {
+    StorytellerUiTool getStoryTellerUiTool();
+  }
+
   public static void main(String[] args) {
 
     checkInputArgs(args);
 
+    StorytellerUiTool uiTool = DaggerStorytellerUiTool_StorytellerUiToolComponent.create().getStoryTellerUiTool();
+
     EventQueue.invokeLater(
         () -> {
-          StorytellerUiTool uiTool = new StorytellerUiTool();
           uiTool.mode = getMode(args[0]);
           uiTool.init();
         });
   }
 
-  private StorytellerUiTool() {}
+  @Inject
+  StorytellerUiTool(Storyteller storyteller, StorytellerConfig storytellerConfig) {
+    this.storyteller = storyteller;
+    this.storytellerConfig = storytellerConfig;
```

Comment:
Maybe you can store `storytellerConfig.getConfig()` config here, like in `StoryWriter`?